src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/MetadataTests/MetadataTests.JsonMetadataServices.cs

```diff
@@ -1,308 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'm deleting these tests because we have enough coverage from the tests in `System.Text.Json.SourceGeneration.Tests.dll`, especially since we've been subjecting the generator to the same tests as the serializer.